client/src/components/participant-details/track-graduation.js

```diff
@@ -100,7 +100,6 @@ export const TrackGraduation = (props) => {
                       : cohortEndDate,
                 },
                 continue: 'continue_yes',
-                withdraw: false,
```

Comment:
why was this removed? if they choose continue_yes, the participant should not be withdrawn from the program (only from the cohort)

Reply:
This is redundant info, we are tracking withdraw in different table.

Reply:
This variable is a form state-var to store previously implemented checkbox.

Reply:
Gotcha!